mvp/app/src/main/java/com/example/mvpexample/Model/User.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.mvpexample.Model
+
+import android.text.TextUtils
+import android.util.Patterns
+import org.w3c.dom.Text
+
+class User(override val email:String, override val password:String) : IUser{
```

Comment:
```suggestion
class User(override val email:String, override val password:String) : IUser {
```

hoxy,, Reformat Code 하셨습니까??
알고계실 것 같지만 윈도우 ctrl + alt + L, 맥 option + command + L 입니당

Reply:
안했습니다.. 냅다 .. 코드를 쓰기만 했네여...